tests/features.py

```diff
@@ -0,0 +1,42 @@
+from flask import current_app
+
+
+def enable_feature(name, value=True):
```

Comment:
FWIW, the name of the function doesn't work for me, because you might end up writing `enable_flag('WOBBLE', False)`, and that reads quite badly.

Maybe `set_flag` would be better?